code/GlobalNavControllerExtension.php

```diff
@@ -6,7 +6,7 @@ public function onBeforeInit() {
 		$host = GlobalNavSiteTreeExtension::get_toolbar_hostname();
 		if (
 				(isset($_REQUEST['regenerate_nav']) || isset($_REQUEST['flush'])) && 
-				$host == Director::absoluteBaseURL() && 
```

Comment:
The reason this broke on local was that //localhost/ssorg was appearing in the global-nav.html and being compared to the local hostname (//localhost) and of course wasn't matching.